src/plugin.ts

```diff
@@ -83,7 +83,7 @@ const createGenerator = (manifest: any) => {
 /**
  * Creates better types for LitElement
  */
-export default {
+export const plugin = {
```

Comment:
Perhaps, instead of `plugin`, it could be `betterLitTypes` so that it would enable:
```typescript
import { betterLitTypes } from 'cem-plugin-better-lit-types'
```

Reply:
Yeah, make more sense